Add some user and tech doc (#3)
vgramer/opa-idea-plugin: docs/user/faq.md

# Table of Content
<!-- toc -->
- [Tests](#tests)

>   - [Why test duration is equal to Zero.](#why-test-duration-is-equal-to-zero)

Reviewer: ```suggestion
  - [Why is the test duration equal to Zero?](#why-is-the-test-duration-equal-to-zero)
```

>   - [Why i dont see pass tests](#why-i-dont-see-pass-tests)

Reviewer: ```suggestion
  - [Why does the plugin not display PASS tests?](#why-does-the-plugin-not-display-pass-tests)
```

<!-- /toc -->

*to update toc, please read [this page](../../hack/README.md).*

# Tests

> ##  Why test duration is equal to Zero.

Reviewer: ```suggestion
##  Why is the test duration equal to Zero?
```

> If you don't make external call (eg http call), evaluate and test a policy is really quick (few hundreds of microseconds)

Reviewer: ```suggestion
If you don't make an external call (eg http call), the evaluation and test of a policy is really quick (few hundreds of microseconds).
```

> but `IntelliJ` store duration with millisecond precision. Consequently if your test take less than a millisecond to be

Reviewer: ```suggestion
But `IntelliJ` stores duration with millisecond precision. Consequently, if your test take less than a millisecond to be
```

> executed it will be show as `0` in the interface.

Reviewer: ```suggestion
executed, it will be shown as `0` in the interface.
```

> Never the less you can see the test duration in the output console.

Reviewer: ```suggestion
Nevertheless, you can see the raw test duration in the output console.
```

> ## Why i dont see pass tests

Reviewer: ```suggestion
## Why does the plugin not display PASS tests?
```

> By default IntelliJ do not show pass tests. To show them you have to click on the "check" icon.

Reviewer: ```suggestion
By default `IntelliJ` does not show pass tests. To show them you have to click on the "check" icon.
```
